cli/dcoscli/analytics.py

```diff
@@ -107,7 +107,7 @@ def _segment_request(path, data):
         requests.post('{}/{}'.format(SEGMENT_URL, path),
                       json=data,
                       auth=HTTPBasicAuth(key, ''),
-                      timeout=1)
```

Comment:
I don't think we need to change it in this PR. I think #243 is already doing this.